Intersect.Client.Framework/Entities/ICritter.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Intersect.Client.Framework.Maps;
+using System;
+using System.Collections.Generic;
+
+namespace Intersect.Client.Framework.Entities
+{
+    public interface ICritter
```

Comment:
`ICritter` should extend `IEntity` and should not redeclare any duplicate properties.

Reply:
I think I didn't do this on purpose because the critter class itself doesn't implement like 90% of the entity class. But I'll look into it.